composite-actions/frontend-generic/prod-deploy/action.yaml

```diff
@@ -0,0 +1,94 @@
+name: 'Deploy prod'
+description: 'deploy prod'
+inputs:
+  SECRET_AUTH:
+    description: 'GCP Auth'
+    required: true 
+  GCLOUD_AUTH_STAGING:
+    description: 'GCP Auth (staging)'
+    required: true
+  GCLOUD_AUTH_PROD:
+    description: 'GCP Auth (prod)'
+    required: true
+  GITHUB_TOKEN:
+    description: 'GH token to create a release'
+  LD_CLIENT_ID:
+    description: 'LaunchDarkly client ID in Secret Manager'
+    required: true
+  IMAGE:
+    description: 'Image name for the service (without tag)'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v1
+
+    - name: Auth to Nexus npm registry
+      uses: extenda/actions/nexus-auth-npm@v0
+      with:
+        service-account-key: ${{ inputs.SECRET_AUTH }}
+
+    - uses: extenda/actions/setup-gcloud@v0
+      id: gcloud
+      with:
+        service-account-key: ${{ inputs.GCLOUD_AUTH_STAGING }}
+
+    - uses: extenda/actions/gcp-secret-manager@v0
+      with:
+        service-account-key: ${{ inputs.GCLOUD_AUTH_PROD }}
+        secrets: |
+          LD_CLIENT_ID: ${{ inputs.LD_CLIENT_ID }} 
+
+    - name: Build and push Docker
+      shell: bash
+      run: |
+        gcloud --quiet auth configure-docker
+        docker build -t ${{ inputs.image }}:${{ github.sha }} .
```

Comment:
We need to pass LD_CLIENT_ID as build arg to Docker

Reply:
done